vendor/github.com/conductorone/dpop/integrations/dpop_grpc/buf.yaml

```
@@ -0,0 +1,12 @@
+version: v1
+breaking:
+  use:
+    - FILE
+lint:
+  use:
+    - DEFAULT
+  except:
+    - PACKAGE_VERSION_SUFFIX
+    - PACKAGE_DIRECTORY_MATCH
+deps:
+  - buf.build/googleapis/googleapis 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)